README.md

```diff
@@ -289,6 +289,11 @@ To find information about supported submodules and there version in SDE please v
 | **GET:- localhost:8080/api/legal-entities** |This API is used to fetch legal entities (list of company's) for Process| Refer Api Doc |Refer Api Doc|
 | **POST:- localhost:8080/api/connectors-discovery** |This API is used to fetch connectors information | Refer Api Doc |Refer Api Doc|
 
+## Detailed API specs available under:
+https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend/blob/main/modules/sde-core/src/main/resources/sde-open-api.yml
+
```

Comment:
I would use relative link: [Detailed API spec](modules/sde-core/src/main/resources/sde-open-api.yml). It us universal and does not depend on repo

Reply:
Done. Please check once.